### ISSUE TITLE
Support caching arbitrary container registries

### ISSUE THREAD

**Issue by ajaffie** [Azure/acr#796]
**What is the problem you're trying to solve**
I'm trying to setup cache rules for all of my dependencies. Only one is currently unsupported, which is `oci.external-secrets.io`.

**Describe the solution you'd like**
It would be nice if a cache rule could be setup for any arbitrary registry that can be used without authentication.
Support for `oci.external-secrets.io` is acceptable as well, considering it is probably fairly widely used.

**Comment by JXavierMSFT** [comment]
Hey @ajaffie, 

Thank you for this feature request. We are evaluating this I will update this thread when we've made a decision.

**Comment by waterfoul** [comment]
Please support this! We have an internal registry we would like to use with ACR as a pull through cache

**Comment by frank-m** [comment]
Yeah this feature would really help with internal registries and also software that has their own registry.
